pkgs/development/tools/misc/ptags/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub
+, cargo
+, lib
+, rustPlatform
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ptags";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "dalance";
+    repo = "ptags";
+    rev = "v${version}";
+    sha256 = "1xr1szh4dfrcmi6s6dj791k1ix2zbv75rqkqbyb1lmh5548kywkg";
+  };
+
+  cargoSha256 = "1rsnb4kzfb577xw7jk0939n42sv94vvspvbz783bmpy9vl53i38k";
+
+  # cargoBuildFlags = lib.optionals stdenv.isLinux [ "--target=x86_64-unknown-linux-musl" ]
+  #   ++ lib.optionals stdenv.isDarwin [ "--target=x86_64-apple-darwin" ];
+
```

Comment:
```suggestion
```

Reply:
Hi @jonringer :)
This has been left this intentionally, as this is a WIP, to illustrate my response to this thread (above): I would like to uncomment it, but then the build fails.